nodeup/pkg/model/logrotate.go

```diff
@@ -35,18 +35,15 @@ type LogrotateBuilder struct {
 var _ fi.ModelBuilder = &LogrotateBuilder{}
 
 func (b *LogrotateBuilder) Build(c *fi.ModelBuilderContext) error {
-	if b.Distribution == distros.DistributionCoreOS {
-		glog.Infof("Detected CoreOS; won't install logrotate")
-		return nil
-	}
-
 	if b.Distribution == distros.DistributionContainerOS {
 		glog.Infof("Detected ContainerOS; won't install logrotate")
 		return nil
+	} else if b.Distribution == distros.DistributionCoreOS {
+		glog.Infof("Detected CoreOS; won't install logrotate")
```

Comment:
How are we enabling?  Confused ... sorry

Reply:
Logrotate is already installed on CoreOS, so we don't need to install the package, only the logrotate conf.

https://github.com/coreos/init/blob/9b0971c8f7f1d634daa399b74f2591cac97f1ede/systemd/system/logrotate.service#L5

https://github.com/coreos/init/blob/9b0971c8f7f1d634daa399b74f2591cac97f1ede/configs/logrotate.conf#L18-L19

Would `c.AddTask(&nodetasks.Package{Name: "logrotate"})` be okay to run away on CoreOS?

Reply:
@julianvmodesto you can answer 

> Would c.AddTask(&nodetasks.Package{Name: "logrotate"}) be okay to run away on CoreOS?

Better than I :)  What do you recommend?

Reply:
No - you generally can't run the Package task on CoreOS - it'll complain that it doesn't know the package manager, because it only supports apt & yum.